examples/nlp/language_modeling/tuning/conf/megatron_gpt_qat_config.yaml

```diff
@@ -0,0 +1,203 @@
+name: llama2-7b
+
+trainer:
+  devices: 1
+  accelerator: gpu
+  num_nodes: 1
+  precision: bf16
+  logger: False # logger provided by exp_manager
+  enable_checkpointing: False
```

Comment:
```suggestion
  enable_checkpointing: True
```

Reply:
This needs to be set to False since `exp_manager.create_checkpoint_callback` is already set to True